src/rpc/blockchain.cpp

```diff
@@ -1531,9 +1531,28 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
     return ret;
 }
 
+UniValue dumpmempool(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() > 0)
+        throw std::runtime_error(
+            "dumpmempool\n"
+            "\nDumps to disk the mempool.\n"
+            "\nExamples:\n"
+            + HelpExampleCli("dumpmempool", "")
+            + HelpExampleRpc("dumpmempool", "")
+        );
+
+    if (!DumpMempool()) {
+        throw JSONRPCError(RPC_INTERNAL_ERROR, "Unable to dump mempool to disk");
+    }
+
+    return "Mempool dumped";
```

Comment:
Because this is machine to machine communication, use `true` as ret val at this point?